tests/services/test_file_svc.py

```diff
@@ -10,6 +10,14 @@
 )
 class TestFileService:
 
+    def test_save_file(self, loop, file_svc, tmp_path):
+        filename = "test_file.txt"
+        payload = b'These are the file contents.'
+        # save temporary test file
+        loop.run_until_complete(file_svc.save_file(filename, payload, tmp_path, encrypt=False))
+        file_location = tmp_path / filename
+        assert os.path.isfile(file_location)
```

Comment:
This is good, I would also verify that the contents of the file match what the input payload is.